core/src/main/java/me/mykindos/betterpvp/core/client/punishments/types/BuildLockPunishmentType.java

```diff
@@ -12,4 +18,19 @@ public String getChatLabel() {
         return "build locked";
     }
 
+    @Override
+    public void onReceive(Client client, Punishment punishment) {
+        Player player = Bukkit.getPlayer(client.getUniqueId());
+        if (player != null) {
+            UtilMessage.message(player, "Punish", "You have been build locked for");
```

Comment:
'for' <insert something here?>

Reply:
Removed, not actually needed since they are notified in chat through punish add.